explore/src/main/scala/explore/services/OdbApiImpl.scala

```diff
@@ -10,7 +10,7 @@ import explore.utils.ToastCtx
 import lucuma.schemas.ObservationDB
 import org.typelevel.log4cats.Logger
 
-case class OdbApiImpl[F[_]: Async]()(using
+case class OdbApiImpl[F[_]: Async](resetCache: String => F[Unit])(using
```

Comment:
is that function beig used?

Reply:
Not yet, but it will. I sent this cutoff PR to keep the size manageable.